### DISCUSSION TITLE
Cant reach server - a little help please.

### DISCUSSION THREAD

**Discussion by audas** [TomBursch/kitchenowl#550] [General]
Edit:

Unknown fix. Sorry.

I went back to the first installation type - have not changed anything and it is working. This is the most frustrating type of error to debug - when it works. Not sure what to do here. Spent a lot of time (an hour, so, not a lot) a bit of time. Wish I knew what this was. Possibly a volumes thing - but also I removed the environment variables so this may also have been something.

Also added a slash to the end of the FRONT_URL="http://144.6.xx.11:8084/"

The third option did throw up a heap of Python warnings and problems.

--
====================================================================

Original
+++++++++

Installed self hosted on a VM machine via all 3 methods. First one gave me a "Can't reach server" page, third all in one method refused to connect - 2nd one had syntax error.

So basically I am definitely reaching the container - but it is throwing me out for some reason.
Just an FYI - I am not putting it through a reverse proxy (I will be using apache2) - I am just trying to access this directly and no dice.

Definitely think its something to do with the config requirements.
 
I am behind a static IP 144.6.xx.11   - I did not put this in as FRONT_END on first docker example - couldnt reach server - put it , in the next 2 versions and refused to connect.

No https at the moment.
Port 8084 is allowed through firewall

I am connected with a port say 8084 so 144.6.xx.11:8084 - should this be in the FRONT_URL?
Do I need the BACK_URL?
Do I need to install KitchenOwl-web or is this now gone?

Edit:

No matter what I put into "FRONT_URL" I get the same "redirection" notice to "http://144.6.xx.11:8084/unreachable"

Thanks for any tips or advice to get this up and running.

**Comment by jhillix** [comment]
Any updates on this? I'm also having the same issue.
